var/spack/repos/builtin/packages/hdf5-vol-async/package.py

```diff
@@ -40,6 +33,7 @@ class Hdf5VolAsync(CMakePackage):
     def setup_run_environment(self, env):
         env.prepend_path("HDF5_PLUGIN_PATH", self.spec.prefix.lib)
         env.set("MPICH_MAX_THREAD_SAFETY", "multiple")
+        env.set("HDF5_VOL_CONNECTOR", "async under_vol=0;under_info={}", raw=True)
```

Comment:
This line existed before but was removed.

Please see the following for the reason:
 https://github.com/spack/spack/pull/33393#issuecomment-1291051774

Reply:
I do not see the point to install something not enabling it. If some Spack package depends on `hdf5-vol-async` does it means user have to set the environment variable manually to have its dependent package working?

Anyway, my goal here is to provide the ability to define environment variable containing `{}`, not to fix (or break) hdf5 packages. So I have removed the commit on `hdf5-vol-async` package.